paymentsheet/src/main/java/com/stripe/android/customersheet/ui/CustomerSheetScreen.kt

```diff
@@ -244,7 +246,7 @@ internal fun AddPaymentMethodWithPaymentElement(
         }
 
         PrimaryButton(
-            label = viewState.primaryButtonLabel,
+            label = viewState.primaryButtonLabel.resolve(context),
```

Comment:
There‘s an extension method that doesn’t require `context`.

Reply:
I'll update it in the next PR